### DISCUSSION TITLE
How do I turn off the Discord Integration?

### DISCUSSION THREAD

**Discussion by zghwi** [neovide/neovide#1465] [Q&A]
Hi, I would like to turn off the Discord RPC.
How do I do that?

**Comment by MultisampledNight** [comment]
Thanks for the question! Though Neovide by itself doesn't have any kind of Discord RPC. Very likely your configuration however does, is it possible that you use a prebuilt configuration?

**Reply by zghwi** [reply]
Oh yeah, I use a custom VimRC, it probably has a Discord RPC.
Anyways thanks!

**Reply by abd0-omar** [reply]
@zplusfour Could you please provide me with the name of the Vimrc plugin for Discord presence?
